### PR TITLE
Deploy scripts.

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+ARG work_dir=/go/src/github.com/Bnei-Baruch/galaxy-monitor
+
+FROM golang:1.14-alpine3.11 as build
+
+LABEL maintainer="bbfsdev@gmail.com"
+
+ARG work_dir
+
+ENV GOOS=linux \
+	CGO_ENABLED=0 \
```
Comment (edoshor) [inline Dockerfile]: remove trailing \\

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+RUN apk update && \
+    apk add --no-cache \
+    git
+
+WORKDIR ${work_dir}
+COPY . .
+
+RUN go test $(go list ./... | grep -v /models) \
```
Comment (edoshor) [inline Dockerfile]: redundant grep. you don't have models folder

File: Dockerfile (hunk 1 above)
Comment (bbfsdev) [reply Dockerfile]: Done

File: Dockerfile (hunk 2 above)
Comment (bbfsdev) [reply Dockerfile]: Done.

**Pull request merged**